tests/platforms/kubernetes/appmanager.go

```diff
@@ -428,6 +454,58 @@ func (m *AppManager) GetHostDetails() (string, string, error) {
 	return podList.Items[0].GetName(), podList.Items[0].Status.PodIP, nil
 }
 
+// SaveContainerLogs get container logs for all containers in the pod and saves them to disk
+func (m *AppManager) SaveContainerLogs() error {
```

Comment:
if possible please add tests for the same.

Reply:
Fair, though tests for our tests is kind of meta. :)

Reply:
I accept that tests for tests is meta :) But just saw that appmanager_test.go exists. So only asked :)

Reply:
No you're right, no reason not to test this.

Reply:
Ok, so I went down this road. To test SaveContainerLogs using the kubernetes client mocks, we need the v0.19.0 golang kubernetes client. Specifically we need https://github.com/kubernetes/client-go/commit/cacbf4f9e0c9bfc9255741e40eafa96c630e26b5 . Upgrading broke a bunch of other things, so I don't think it is worth spending any more time trying to unit test part of our e2e test fixture.

Reply:
we can revisit later... :) investigating e2e test is more important now...